src/components/patch/PatchContainer.tsx

```diff
@@ -39,7 +43,15 @@ export class PatchContainer extends React.Component<Props, State> {
 
     return (
       <div>
-        <div className="patch-search-bar">Insert patch search bar here</div>
+        <div className="search-container">
+          <Paper>
+            <InputBase startAdornment={<IconButton><SearchIcon /></IconButton>}
```

Comment:
Since this searches as you type, we should just have an icon here rather than a button

Reply:
Fixed (I thought the icon button was required for an icon for some reason)